playbooks/roles/docker/tasks/RedHat.yml

```diff
@@ -16,28 +16,9 @@
     gpgcheck: no
   when: docker_mirror_url is defined
 
-# docker-ce-18 requries pigz which is available starting from rhel8
-# so, use epel
-- name: configure epel
-  yum:
-    name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{ ansible_distribution_major_version }}.noarch.rpm
-  when: ansible_distribution_major_version|int < 8
```

Comment:
@rajujith 
I believe this works in rocky8. will deployment on centos7 be broken if these steps are removed ?